suitcase/hdf5.py

```diff
@@ -17,14 +17,29 @@
 
 __version__ = "0.2.2"
 
-def export(headers, filename):
+def export(headers, filename, stream_name=None, fields=None, timestamps=True, use_uid=True):
     """
+    Create hdf5 file to preserve the structure of databroker.
+
     Parameters
     ----------
     headers : a Header or a list of Headers
         objects retruned by the Data Broker
     filename : string
         path to a new or existing HDF5 file
+    stream_name : string, optional
+        None means save all the data from each descriptor, i.e., user can define stream_name as primary,
+        so only data with descriptor.name == primary will be saved.
+        The default is None.
+    fields : list, optional
+        whitelist of names of interest; if None, all are returned;
+        This is consistent with name convension in databroker.
+        The default is None.
+    timestamps : Bool, optional
+        save timestamps or not
+    use_uid : Bool, optional
+        Create group name at hdf file based on uid if this value is set as True.
+        Otherwise group name is created based on beamline id and run id.
```

Comment:
ok, that might be a bit strong (I am apparently cranky this morning) and understand that the ship has sailed on this.